app/front/scripts/services/download-package.js

```diff
@@ -80,8 +80,8 @@
               StorageService.clearApplicationState()
                   .then(function() {
                     var packageName = PackageService.getAttributes().name;
-                    var owner = LoginService.email;
-                    state.packagePublicUrl = '/viewer/'+owner+':'+packageName; //dataPackage.uploadUrl;
+                    var owner = LoginService.userid;
```

Comment:
Match on email, as we do in the Open Data Census https://github.com/okfn/opendatacensus/blob/master/census/routes/utils.js#L22